docs/docsite/rst/guide_vmware.rst

```diff
@@ -0,0 +1,62 @@
+Getting Started with VMware
+===========================
+
+Introduction
+````````````
+
+Ansible provides various modules to manage VMware infrastructure which includes datacenter, cluster,
+host system and virtual machine etc.
+
+Requirements
+````````````
+
+Ansible VMware modules are written on top of `pyVmomi <https://github.com/vmware/pyvmomi>`_.
+pyVmomi is the Python SDK for the VMware vSphere API that allows user to manage ESX, ESXi,
+and vCenter infrastcture. User can install pyVmomi using
+
+.. code-block:: bash
+
+    $ pip install pyvmomi
+
+
+vmware_guest module
+```````````````````
+
+The ``vmware_guest`` module is used to manage various operations related to virtual machines in
```

Comment:
`The :ref:`vmware_guest <vmware_guest>` module...`

Reply:
ok.